code/modules/shuttle/medisim.dm

```diff
@@ -0,0 +1,55 @@
+///These are for the medisim shuttle
+
+/obj/machinery/capture_the_flag/medisim
+	game_area = /area/shuttle/escape/simulation
+
+/obj/machinery/capture_the_flag/medisim/spawn_team_member(client/new_team_member)
+	var/mob/living/carbon/human/M = ..()
+	M.remove_all_languages(LANGUAGE_CTF)
+	M.grant_language(/datum/language/monkey, TRUE, TRUE, LANGUAGE_CTF)
```

Comment:
Named parameters, please. `TRUE, TRUE` is not obvious what it means without looking into the implementation of `grant_language`.